app/src/main/java/de/xikolo/managers/DownloadManager.kt

```diff
@@ -231,7 +234,8 @@ class DownloadManager(activity: FragmentActivity) {
     }
 
     fun downloadRunning(downloadAsset: DownloadAsset): Boolean {
-        return DownloadService.getInstance()?.isDownloading(downloadAsset.url) == true
+        return downloadAsset.url != null
```

Comment:
see the comment about this in `DownloadService`

Reply:
Same as above, please double check.